sdk/documenttranslation/Azure.AI.Translator.DocumentTranslation/tests/Infrastructure/DocumentTranslationLiveTestBase.cs

```diff
@@ -15,7 +15,7 @@ namespace Azure.AI.Translator.DocumentTranslation.Tests
 {
     public class DocumentTranslationLiveTestBase : RecordedTestBase<DocumentTranslationTestEnvironment>
     {
-        protected TimeSpan PollingInterval => TimeSpan.FromSeconds(Mode == RecordedTestMode.Playback ? 0 : 30);
+        protected TimeSpan PollingInterval => TimeSpan.FromSeconds(Mode == RecordedTestMode.Playback ? 0 : 2);
```

Comment:
already discussed this offline, but maybe update to 30 for live until we know more?